packages/cozy-konnector-libs/src/libs/request/request.spec.js

```diff
@@ -144,16 +144,23 @@ describe('requestFactory', () => {
       try {
         await rq('https://untrusted-root.badssl.com')
       } catch (err) {
-        expect(err.message).toEqual(
-          'Error: self-signed certificate in certificate chain'
+        // Second string matching is for node 16 retro compatibility
+        expect(err.message).toMatch(
+          /Error: self-signed certificate in certificate chain|Error: self signed certificate in certificate chain/
         )
       }
     })
 
     test('Self-signed cert should be refused', async () => {
-      return expect(rq('https://self-signed.badssl.com')).rejects.toEqual(
-        new Error('Error: self-signed certificate')
-      )
+      if (process.version.startsWith('v16')) {
```

Comment:
I cannot see any difference between the two alternatives

Reply:
Ok, let's add a comment on this to avoid to have someone remove it later because he/she does not see the point.